### PR TITLE
added stream transaction issues to known issues

### PR DESCRIPTION
**Pull request by jsteemann**
Add stream transaction limitations/issues to the list of known issues for 3.5.

### CODE REVIEW

File: 3.5/release-notes-known-issues35.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ Upgrading
...
 
+Stream Transactions
+-------------------
+| Issue      |
+|------------|
+| **Date Added:** 2019-08-19 <br> **Component:** Transactions <br> **Deployment Mode:** All <br> **Description:** Stream Transactions currently do not honor the limits described in the documentation. Currently the idle timeout of 10 seconds will not be enforced, neither will the maximum size of transaction be enforced. It is planned to add the enforcements of the limits with ArangoDB 3.5.1. <br> **Affected Versions:** 3.4.0RC1 to 3.4.0 (including) <br> **Fixed in Versions:** - <br> **Reference:** N/A |
+| **Date Added:** 2019-08-19 <br> **Component:** Transactions <br> **Deployment Mode:** All <br> **Description:** Stream Transactions currently do not support the graph operations that are initiated via the `general-graph` JavaScript module or via the REST API at `/_api/gharial`. Right now these operations will act as if no stream transaction is present. Stream transaction support for these operations will be added later. <br> **Affected Versions:** 3.4.0RC1 to 3.4.0 (including) <br> **Fixed in Versions:** - <br> **Reference:** N/A |
```
Comment (joerg84) [inline 3.5/release-notes-known-issues35.md]: Is the affected version correct @jsteemann

Comment (KVS85) [reply 3.5/release-notes-known-issues35.md]: Fixed.

**Comment by jsteemann** [comment]
Yes. But it currently allows different users to see and modify each others transactions, which is not great

**Comment by Simran-B** [comment]
https://deploy-preview-141--zealous-morse-14392b.netlify.com/docs/3.5/release-notes-known-issues35.html#stream-transactions

Table markup appears to be broken, will fix it

**Pull request merged**